### PR TITLE
I5045 change documentation theme

### PR DESCRIPTION
**Pull request by ABSitf**

### CODE REVIEW

File: Doxyfile [hunk 1]
```diff
@@ -1388,7 +1395,7 @@ HTML_EXTRA_FILES       =
 # The default value is: AUTO_LIGHT.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_COLORSTYLE        = AUTO_LIGHT
+HTML_COLORSTYLE        = LIGHT
```
Comment (Grantim) [inline Doxyfile:1398]: can we use auto theme?

File: Doxyfile [hunk 2]
```diff
@@ -1365,7 +1365,10 @@ HTML_STYLESHEET        =
 # documentation.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_EXTRA_STYLESHEET  = ../MeshLib/doxygen/CustomStyle.css
+HTML_EXTRA_STYLESHEET  = ../MeshLib/doxygen/CustomStyle.css \
```
Comment (Grantim) [inline Doxyfile:1368]: do we still need our css file?

**Review by Grantim — APPROVED** [review]

File: Doxyfile (hunk 2 above)
Comment (ABSitf) [reply Doxyfile:1368]: yes

File: Doxyfile (hunk 1 above)
Comment (ABSitf) [reply Doxyfile]: updated

**Pull request merged**